worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -978,7 +964,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
           callback.onFailure(new CelebornIOException(StatusCode.REGION_FINISH_FAIL_SLAVE, e))
         case _ =>
           workerSource.incCounter(WorkerSource.PushDataFailCount)
-          callback.onFailure(new CelebornIOException(StatusCode.PUSH_DATA_FAIL_SLAVE, e))
+          callback.onFailure(new CelebornIOException(StatusCode.PUSH_DATA_WRITE_FAIL_SLAVE, e))
```

Comment:
I think we should check if e is CelebornIOException, if yes just use it, if not, use PUSH_DATA_FAIL_SLAVE, we can't be sure that it's PUSH_DATA_WRITE_FAIL_SLAVE here

Reply:
Done